### ISSUE TITLE
AWS access for Ian Harrison

### ISSUE THREAD

**Issue by it-harrison** [department-of-veterans-affairs/va.gov-team#45329]
### COR Name

Vilay Senthep

### Vendor Onboarding Representative Name

Oddball - Amber Malcolm

### Your Name

Ian Harrison

### Your Email

ian.harrison@adhocteam.us

### Team, Role, and Company of the target individual

Release Tools Team, Software Engineer III, Ad Hoc

### Product Manager (PM) name and email

Lainey Trahan lainey.trahan@adhocteam.us

### Product Owner (PO) name and email

Micheal Chelen michael.chelen@va.gov

### Desired AWS Access

I work on vets-website. Right now I am trying to ssh into Jenkins to monitor the build of my review instance, but am unable due to MFA requirement on AWS. Probably you could copy the permissions of team-mates Theo Bentum or Chris Alexander. Thanks!

### Access Expiration

ongoing

### Additional Notes

I expect to occasionally need to look at Review Instance builds or assets in the vets-website buckets as a member of the Release Tools Team

### User must exist in a roster before AWS access can be granted

- [ ] Search for user on the VFS Team Roster: https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665
- [x] Or search for user on the Platform Team Roster: https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster
- [ ] If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the Platform orientation process https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html
- [ ] If a user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added.
- [x] Comment in this issue saying which roster the user is listed in.

**Comment by calquisola** [comment]
@it-harrison is part of Platform Release Tools Team.

**Comment by calquisola** [comment]
For AWS ACCESS
User: Ian Harrison
COR: Vilay Senthep
VOR: Amber Malcolm
Approval needed: @department-of-veterans-affairs/operations-aws-approvers 
AWS Admin Role: no

**Comment by td-usds** [comment]
Reached out to VOR

**Comment by td-usds** [comment]
@it-harrison  Are you requesting to turn off the MFA requirement?

**Comment by it-harrison** [comment]
> @it-harrison Are you requesting to turn off the MFA requirement?

no, I understood that I needed AWS access to SSH into a Review Instance.

**Comment by td-usds** [comment]
Your request doesn't mention a review instance though, it asks for the ability to ssh into Jenkins and being prevented from doing so

> I work on vets-website. Right now I am trying to ssh into Jenkins to monitor the build of my review instance, but am unable due to 

Can you elaborate on what the exact scenario that you're trying to accomplish? Maybe there's another way to get you what you need...

**Comment by it-harrison** [comment]
> 

My problem was that a Review Instance was not building and I wanted to check the logs in the container; when I tried to SSH into the review instance as specified in the [platform docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/using-review-instances-to-preview-changes#Usingreviewinstancestopreviewchanges-SSH) the process failed because (I was told) I lacked AWS access for MFA. I was able to resolve the original problem by rebuilding the Review Instance through Jenkins. So I don't have any immediate need for AWS, more of anticipating a future need.
